decibel/theme.json

```diff
@@ -145,7 +152,44 @@
 			"heading": {
 				"typography": {
 					"fontFamily": "var(--wp--preset--font-family--rubik)",
-					"textTransform": "uppercase"
+					"fontWeight": "700",
+					"textTransform": "uppercase",
+					"lineHeight": "1.6"
+				}
+			},
+			"h1": {
+				"typography": {
+					"fontSize": "clamp(3rem, 1.6153846153846154rem + 2.8846153846153846vw, 4.5rem)",
+					"lineHeight": "1.1"
+				}
+			},
+			"h2": {
+				"typography": {
+					"fontSize": "clamp(2rem, 0.15384615384615374rem + 3.8461538461538463vw, 4rem)",
+					"lineHeight": "1.1"
+				}
+			},
+			"h3": {
+				"typography": {
+					"fontSize": "clamp(1.75rem, 1.2884615384615383rem + 0.9615384615384616vw, 2.25rem)",
```

Comment:
Can we use `preset--x-large` here?

Reply:
Oh good call, done here: https://github.com/Automattic/themes/pull/6571/commits/659afa987b73334fae3c5164653647eb17b4b0fe